common/v2/features/Dashboard/components/WalletBreakdown/AccountDropdown.tsx

```diff
@@ -94,8 +95,18 @@ const SCheckbox = styled(Checkbox)`
   &:hover {
     background-color: ${GREY_LIGHTEST};
   }
+
+  > label {
+    min-width: 20px;
+  }
+
+  img {
+    min-width: 30px;
+  }
 `;
 
+const trimAccountLabel = (label: string) => (label.length > 65 ? `${label.substr(0, 65)}…` : label);
```

Comment:
Please
- create utility function which accepts a dynamic length (to encourage reuse).
- add simple unit test (respects length, handles undefined, includes ellipsis) to facilitate maintenance

PS: We're going to start focusing on increasing our code and case coverages. Abstracting utility functions and uniting testing when possible is a good place to start.